src/shared/layouts/BaseLayout/components/TopBar/Breadcrumbs/Breadcrumbs.hooks.ts

```diff
@@ -122,21 +122,20 @@ export const useBreadcrumbs = (restCrumbs?: Breadcrumb[]) => {
           : page.dashboardManagers,
       });
     }
-    if (pathname.includes('respondents')) {
-      newBreadcrumbs.push({
-        icon: 'respondent-outlined',
-        label: t('respondents'),
-        navPath: appletId
-          ? generatePath(page.appletRespondents, { appletId })
-          : page.dashboardRespondents,
-      });
-    }
+
+    // TODO: validate if this would be removed
```

Comment:
This is needed for Data Visualization and Individual Schedule, at least for now.

Reply:
Great, thank you for the clarification